src/test/resources/LuckySearch.feature

```diff
@@ -0,0 +1,9 @@
+Feature: Search
```

Comment:
Tag all scenarios with @Smoke as per task 3 in the readme

Reply:
@Smoke
  Scenario: Relevant search results are returned to the user
    Given I am on the Google UK homepage
    When I enter "BBC" search term
    Then results relevant to the search term are returned